simphony/io/data_container_description.py

```diff
@@ -70,16 +70,23 @@ class Data(tables.IsDescription):
     ln_of_restitution_coefficient = tables.Float64Col(pos=63)
     rolling_friction = tables.Float64Col(pos=64)
     volume_fraction = tables.Float64Col(pos=65)
+    point = tables.Int32Col(pos=66)
```

Comment:
Sorry, a forgot about that. Yes, you make a very good point. How about calling this new `enum` as `CUBAItems `